datasets/aquamuse/README.md

```diff
@@ -0,0 +1,167 @@
+- crowdsourced
+- expert-generated
+languages:
+- en
+licenses: []
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- extended|other-Google-Natural-Questions-and-Common-Crawl
+- original
+task_categories:
+- other
+- question-answering
+task_ids:
+- abstractive-qa
+- extractive-qa
+- other-other-query-based-multi-document-summarization
+---
+
+# Dataset Card for AQuaMuSe
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/google-research-datasets/aquamuse
+- **Repository:** https://github.com/google-research-datasets/aquamuse
+- **Paper:** https://arxiv.org/pdf/2010.12694.pdf
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+AQuaMuSe is a novel scalable approach to automatically mine dual query based multi-document summarization datasets for extractive and abstractive summaries using question answering dataset (Google Natural Questions) and large document corpora (Common Crawl)
+
+This dataset contains versions of automatically generated datasets for abstractive and extractive query-based multi-document summarization as described in [AQuaMuSe paper](https://arxiv.org/pdf/2010.12694.pdf).
+### Supported Tasks and Leaderboards
+
+- **Abstractive** and **Extractive** query-based multi-document summarization
+- Question Answering
+
+### Languages
+
+    en : English
+
+## Dataset Structure
+
+### Data Instances
+
+- `input_urls`: a `list` of `string` features. 
+- `query`: a `string` feature.
+- `target`: a `string` feature
+
+
+Example: 
+
+    {'input_urls': ['https://boxofficebuz.com/person/19653-charles-michael-davis'],
+     'query': 'who is the actor that plays marcel on the originals',
+     'target': "In February 2013, it was announced that Davis was cast in a lead role on The CW's new show The Originals, a spinoff of The Vampire Diaries, centered on the Original Family as they move to New Orleans, where Davis' character (a vampire named Marcel) currently rules."}
```

Comment:
````suggestion
```
{
    'input_urls': ['https://boxofficebuz.com/person/19653-charles-michael-davis'],
     'query': 'who is the actor that plays marcel on the originals',
     'target': "In February 2013, it was announced that Davis was cast in a lead role on The CW's new show The 
Originals, a spinoff of The Vampire Diaries, centered on the Original Family as they move to New Orleans, where 
Davis' character (a vampire named Marcel) currently rules."
}
```
````